### ISSUE TITLE
Not receiving notifications

### ISSUE THREAD

**Issue by Vex-iqualify** [indywidualny/FaceSlim#156]
So this might be a problem with my phone, because I have a Samsung Galaxy S3 US Cellular model, and it doesn't play well with a whole lot of apps, but basically I'm not getting any phone notifications coming from the app, and I'd really like to receive the messenger ones (actually would prefer not to get the others).

Show notifications is enabled in app info, and the settings button doesn't bring up anything. I'm assuming I'm not missing something, because the change logs mention notifications. I'm running 2.4.1.

**Comment by indywidualny** [comment]
Enable logging please and share the file after a few hours. It will be located at the root directory of your external storage.

**Comment by indywidualny** [comment]
Fired - https://github.com/indywidualny/FaceSlim/releases/tag/v2.6.0-beta3
Gonna be released soon.

**Comment by moshpirit** [comment]
Using the last version and have the same problem. The only notifications I got were PMs and the app was open in the background (once I had my phone locked and the others I just was checking other app).

I noticed that it doesn't load the notifications correctly, so even if I update the page it doesn't say anything about new notifications (interactions or friendship notifications).    
So kind of it only notifies the messages.

Using the last version on F-droid on Android 6.0.1
blue bar correction and notifications are enabled.
